### ISSUE TITLE
which was built for iOS 14.4

### ISSUE THREAD

**Issue by FikoKaramac** [ptoomey3/Keychain-Dumper#66]
I used checkra1n to jailbreak the device (iphone 6), after that I ssh into the phone and when trying to execute the executable it spits out this error

`dyld: Symbol not found: _objc_opt_new
  Referenced from: /private/var/tmp/./keychain_dumper (which was built for iOS 14.4)
  Expected in: dyld shared cache
 in /private/var/tmp/./keychain_dumper
Abort trap: 6
-bash-3.2# ./keychain_dumper -h
dyld: Symbol not found: _objc_opt_new
  Referenced from: /private/var/tmp/./keychain_dumper (which was built for iOS 14.4)
  Expected in: dyld shared cache
 in /private/var/tmp/./keychain_dumper
Abort trap: 6`

**Comment by ptoomey3** [comment]
To be fair, I'm not sure what is going on there. I've been out of this space for quite a few years and have been trying to re-build the project with successive iOS relases, but without any actual jailbroken device to test with on my end. It might be worth trying to build from scratch on your end for the target device iOS version you are wanting it to run on.

**Comment by auspicious99** [comment]
I got a similar error on iPhoneXS Max jailbroken with uncover.

`dyld: Symbol not found: _objc_opt_new
  Referenced from: /sbin/./keychain_dumper (which was built for iOS 14.1)
  Expected in: /usr/lib/libobjc.A.dylib
 in /sbin/./keychain_dumper
Abort trap: 6`

**Comment by alyxferrari** [comment]
@FikoKaramac @auspicious99 Install the Xcode version corresponding to your iOS version, run `xcode-select` on it, and compile the project from source.

**Comment by SupaYoshi** [comment]
> 
> 
> @FikoKaramac @auspicious99 Install the Xcode version corresponding to your iOS version, run `xcode-select` on it, and compile the project from source.

@alyxferrari  are you saying that doing it that method, it will work again? Please advice

**Comment by alyxferrari** [comment]
@SupaYoshi Yes, I've had this happen before. Download whichever Xcode version is for your iOS version from [here](https://developer.apple.com/download/more/?=xcode), if you're on iOS 14.5, then just download the latest version of Xcode instead. Download the command-line tools for whichever version you download, too.`xcode-select` only needs to be run if you have more than one version of Xcode on your system. If you do, run it like this:
`sudo xcode-select -s /path/to/Xcode.app`
And of course replace `/path/to/Xcode.app` with wherever the Xcode version you just downloaded is. After that, install Homebrew on your system by running the command at [this site](https://brew.sh), then run `brew install make`. Finally, download the Keychain-Dumper source, move to that directory with `cd` in the terminal, and run `make`. That should make a version of Keychain-Dumper compiled specifically for your iOS version, which will remove the error mentioned in this issue.

**Comment by raulsiles** [comment]
Release v1.1.0 works in iOS 14.x. Also see issue #68.

**Comment by ptoomey3** [comment]
@raulsiles - would you say we can close this out if 1.1.0 works?

**Comment by alyxferrari** [comment]
I think so. I've also gotten it to work on iOS 14 by reducing the entitlements to under 36.
